src/ci/bin/test.inc.sh

```diff
@@ -355,15 +355,11 @@ cromwell::private::create_build_variables() {
     backend_type="${backend_type#centaurWdlUpgrade}"
     backend_type="${backend_type#centaurHoricromtal}"
     backend_type="${backend_type#centaur}"
-    backend_type="${backend_type#conformance}"
     backend_type="$(echo "${backend_type}" | sed 's/\([A-Z]\)/_\1/g' | tr '[:upper:]' '[:lower:]' | cut -c 2-)"
     CROMWELL_BUILD_BACKEND_TYPE="${backend_type}"
 
-    if [[ "${CROMWELL_BUILD_TYPE}" == conformance* ]]; then
-        CROMWELL_BUILD_SBT_ASSEMBLY_COMMAND="server/assembly centaurCwlRunner/assembly"
-    else
-        CROMWELL_BUILD_SBT_ASSEMBLY_COMMAND="assembly"
-    fi
+    # It may be possible to trim this down to e.g. `server/assembly` for some jobs
+    CROMWELL_BUILD_SBT_ASSEMBLY_COMMAND="assembly"
```

Comment:
Interesting opportunity for more optimization!

Reply:
Yeah, it seems to cut down the JAR build from 7 to 4 minutes when all you need is `server`. But it was getting to be confusing making too many changes at once so I deferred the less-impactful changes for later.